product_attribute_value_archive/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+Allows to archive a `product.attribute.value` which is referenced by archived `product.product`.
+
+In std odoo, removing a `product.attribute.value` from a `product.template`
+either leads to a `product.product` unlink or archive,
+depending on if the variant is referenced on a `sale.order.line` or not.
+Then, if user tries to remove the `product.attribute.value` from the `product.attribute`,
+he will get an error because of the `ondelete restrict` on `product.product`.
```

Comment:
here you can mention a PR on odoo in case they merge it master to not migrate in future

Reply:
It will never be merged, I opened this PR so they're aware that this is broken.
The better they can do is to implement it for 16.0 :/

Reply:
High chance they don't do it, perhaps opening a PR on `master` to have better chances...

Reply:
I'll do that later, no time for this right now.

Reply:
yes pls